config/environments/development.rb

```diff
@@ -54,9 +57,10 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  # This not used as evented file updates are not supported via VM
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   # Using polling file checker, requires VM time to be in sync with host
   config.file_watcher = ActiveSupport::FileUpdateChecker
+
+  config.hosts << "content-data-admin.dev.gov.uk"
```

Comment:
I think we've decided the best way to do this is 

```
config.hosts += [ 
  "content-data-admin.dev.gov.uk"
]
```
Also, is there a draft version of this app that needs to be added here too?

Reply:
Cool, i'll update to that 👍 . 
Doesn't appear to be a draft version